internal/services/loganalytics/log_analytics_workspace_resource_test.go

```diff
@@ -276,6 +276,20 @@ func TestAccLogAnalyticsWorkspace_cmkForQueryForced(t *testing.T) {
 	})
 }
 
+func TestAccLogAnalyticsWorkspace_withUseOnlyResourcePermission(t *testing.T) {
```

Comment:
Can we add a step to this and toggle the value? i.e. toggle from true -> false -> true
```suggestion
func TestAccLogAnalyticsWorkspace_ToggleAllowOnlyResourcePermission(t *testing.T) {
```